code/modules/power/supermatter/supermatter_delamination/cascade_delam_objects.dm

```diff
@@ -90,6 +90,8 @@
 	return COMPONENT_CANCEL_ATTACK_CHAIN
 
 /obj/crystal_mass/Destroy()
+	STOP_PROCESSING(SSsupermatter_cascade, src)
```

Comment:
This is fine, but remove the light source list, and I explained over discord, but the problem is likely because of this

https://github.com/tgstation/tgstation/blob/0047063f9784dc4213af6401fd82f6f304773d9c/code/modules/power/supermatter/supermatter_delamination/cascade_delam.dm#L66-L68

crystal masses are being created before SSsupermatter_cascade is set to fire, so you just have to move this 2 lines down.